<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlMainWindow</class>
 <widget class="QMainWindow" name="controlMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>controlMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,6">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="connectWidget" name="connectionWidget">
      <property name="title">
       <string>Connect Tools</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="sensorsWidget" name="sensorViewWidget">
      <property name="title">
       <string>Sensors</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="effectsWidget">
      <property name="title">
       <string>Effects</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>connectWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">connectwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>sensorsWidget</class>
   <extends>QGroupBox</extends>
   <header location="global">sensorswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
